torchvision/csrc/cpu/image/read_write_file_cpu.cpp

```diff
@@ -13,8 +20,17 @@ torch::Tensor read_file(std::string filename) {
 
   TORCH_CHECK(size > 0, "Expected a non empty file");
 
+#ifdef _WIN32
+  torch::Tensor data;
+  {
+    auto data_orig =
+        torch::from_file(filename, /*shared=*/false, /*size=*/size, torch::kU8);
+    data = std::move(data_orig.detach().clone());
```

Comment:
Given that we are cloning the tensor, do we still need the move?
And is the `detach()` necessary given that we have created the tensor right away, so there is no history?

Reply:
@fmassa This `move` is used to avoid calling the copy constructor/assignment in line 28. Do you mean we can actually do `data = torch::from_file(filename, /*shared=*/false, /*size=*/size, torch::kU8).clone()`?

Reply:
Yes, that was my thinking. Is there any benefit from doing as it is currently done?

Reply:
@fmassa Nope. Will change according to your suggestion.